conference.js

```diff
@@ -629,48 +630,65 @@ export default {
             // Resolve with no tracks
             tryCreateLocalTracks = Promise.resolve([]);
         } else {
-            tryCreateLocalTracks = createLocalTracksF(
-                { devices: initialDevices }, true)
-                .catch(err => {
-                    if (requestedAudio && requestedVideo) {
 
-                        // Try audio only...
-                        audioAndVideoError = err;
+            const _createLocalTracks = trackEffects =>
```

Comment:
I'm usually trying to avoid declaring functions inside a function. How about the following implementation:
```
const loadEffectsPromise = options.startWithBlurEnabled 
    ? loadScript('libs/video-blur-effect.min.js').then(() =>
        Object.values(getJitsiMeetGlobalNS().effects))
    .catch(e => {
        logger.error(...); return [];
    }) : Promise.resolve([]);
tryCreateLocalTracks = loadEffectsPromise.then(effects => createLocalTracksF(
                    {
                        devices: initialDevices,
                        effects: trackEffects
                    } ...... and the rest of the code from the function)
```

WDYT?